### PR TITLE
remind: 03.03.05 -> 03.03.06

### PR DESCRIPTION
**Pull request by jshholland**
Also apply nixpkgs-format.

###### Motivation for this change

Update to new package version, see announcement [here](https://dianne.skoll.ca/pipermail/remind-fans/2021/003826.html).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119297` at 034a957b run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>remind</li>
    <li>wyrd</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl2` is a deprecated license, please check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/tools/misc/remind/default.nix:51:5:
```
   |
51 |     license = lib.licenses.gpl2;
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/tools/misc/remind/default.nix [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
+, fetchurl
+, tk
+, tcllib
+, makeWrapper
 , tkremind ? true
-} :
+}:
 
 assert tkremind -> tk != null;
 assert tkremind -> tcllib != null;
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/remind/default.nix:11]: This is not required and does not do anything.

Comment (7c6f434c) [reply pkgs/tools/misc/remind/default.nix:11]: Unless someone uses overrides.

Comment (SuperSandro2000) [reply pkgs/tools/misc/remind/default.nix:11]: You shouldn't just override any random input to null. New packages and updates should use boolean flags instead.

**Pull request merged**